internal/api/models/health.go

```diff
@@ -10,3 +10,9 @@ type HealthCheck struct {
 	Timestamp time.Time
 	Healthy   bool
 }
+
+// NodeConnectionStatus ... Used to display health status of each node connection
+type NodeConnectionStatus struct {
```

Comment:
I feel like this may be a better name:
```suggestion
type BedrockConnectionStatus struct {
```
 Versus 'NodeConnectionStatus' which implies only a single node.

Reply:
Or if we want to be more general, we can call it 'ChainConnectionStatus'